litr/src/main/java/com/linkedin/android/litr/utils/TranscoderUtils.java

```diff
@@ -154,13 +154,15 @@ public static int estimateVideoTrackBitrate(@NonNull MediaSource mediaSource, in
             }
         }
 
+        long videoTrackDuration = TimeUnit.MICROSECONDS.toSeconds(videoTrackFormat.getLong(MediaFormat.KEY_DURATION));
```

Comment:
I think it would be better to convert this to a float to handle fractions of a second. That would make our calculations more precise, too.